osf_tests/test_guid_annotations.py

```diff
@@ -6,29 +6,30 @@
 from osf.models import OSFUser
 from django.db.models import Max, DateTimeField
 
-from osf_tests.factories import UserFactory, PreprintFactory
+from osf_tests.factories import UserFactory, PreprintFactory, NodeFactory
 
 
 @pytest.mark.django_db
 class TestGuidAnnotations:
     guid_factories = [
         UserFactory,
         PreprintFactory,
+        NodeFactory
     ]
 
     @pytest.mark.parametrize('Factory', guid_factories)
     def test_filter_object(self, Factory):
         obj = Factory()
-        assert '_guids__' in str(obj._meta.model.objects.filter(id=obj.id).query), 'Guid annotations did not exist in filter query for {}'.format(obj._meta.model.__name__)
+        assert '__guids' in str(obj._meta.model.objects.filter(id=obj.id).query), 'Guid annotations did not exist in filter query for {}'.format(obj._meta.model.__name__)
```

Comment:
Error message should be something like `"Guids were not included in filter query...."`